Code/GraphMol/FMCS/DebugTrace.h

```diff
@@ -16,6 +16,7 @@
 #include <iostream>
 #ifdef _MSC_VER
 #define _CRT_SECURE_NO_WARNINGS
+#define NOMINMAX
```

Comment:
yeah, because otherwise including `Windows.h` adds a `#define` for `max` which causes the new code not to build.
I'm amazed we managed to make it this long without needing this one.